packages/experiments/src/components/Form/inputs/dropdown/FormDropdown.tsx

```diff
@@ -1,29 +1,46 @@
 import * as React from 'react';
 
 // Components
-import { IFormDropdownProps } from './FormDropdown.types';
+import { IFormDropdownProps, DropdownValue } from './FormDropdown.types';
 import { Dropdown, IDropdownOption, IDropdownProps } from 'office-ui-fabric-react/lib/Dropdown';
 import { FormBaseInput, IFormBaseInputState } from '../../FormBaseInput';
 import { IFormContext } from '../../Form';
 
 /**
  * Dropdown input for Form
  */
-export class FormDropdown extends FormBaseInput<
-  number | string,
-  IFormDropdownProps,
-  IFormBaseInputState<number | string>
-> {
+export class FormDropdown extends FormBaseInput<DropdownValue, IFormDropdownProps, IFormBaseInputState<DropdownValue>> {
   constructor(props: IFormDropdownProps, context: IFormContext) {
     super(props, context);
+
+    const { dropdownProps } = props;
+
+    const propsValue = this.props.value;
+
+    let currentValue: DropdownValue | undefined = undefined;
+
+    if (dropdownProps && dropdownProps.multiSelect) {
+      // If multiSelect is set to true the currentValue should be an array.
+      if (Array.isArray(propsValue)) {
+        currentValue = propsValue;
+      } else if (propsValue !== undefined) {
```

Comment:
Again, here, just use falsy/truthy checks, and everywhere else in this file

Reply:
Changed it, thanks. The places where it remains unchanged is because the value can be the number 0, which would cause problems.